geth/notification/fcm_provider_test.go

```diff
@@ -0,0 +1,81 @@
+package notification
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NaySoftware/go-fcm"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/notification/message"
+	t "github.com/status-im/status-go/geth/testing"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestFCMClientTestSuite(t *testing.T) {
+	suite.Run(t, new(FCMProviderTestSuite))
+}
+
+type FCMProviderTestSuite struct {
+	t.BaseTestSuite
+
+	fcmClientMock     *common.MockFirebaseClient
+	fcmClientMockCtrl *gomock.Controller
+}
+
+func (s *FCMProviderTestSuite) SetupTest() {
+	s.fcmClientMockCtrl = gomock.NewController(s.T())
+	s.fcmClientMock = common.NewMockFirebaseClient(s.fcmClientMockCtrl)
+}
+
+func (s *FCMProviderTestSuite) TearDownTest() {
+	s.fcmClientMockCtrl.Finish()
+}
+
+func (s *FCMProviderTestSuite) TestNewFCMClient() {
+	fcmClient := NewFCMProvider(s.fcmClientMock)
+
+	s.Require().NotNil(fcmClient)
+}
+
+func (s *FCMProviderTestSuite) TestSetMessage() {
+	ids := []string{"1"}
+	body := interface{}("body")
+
+	s.fcmClientMock.EXPECT().NewFcmRegIdsMsg(ids, body).Times(1)
+	fcmClient := NewFCMProvider(s.fcmClientMock)
+
+	fcmClient.SetMessage(ids, body)
```

Comment:
Not sure why you decide to have a new line before this one.

Isn't this one easier to follow?
```go
ids := []string{"1"}
body := interface{}("body")

s.fcmClientMock.EXPECT().NewFcmRegIdsMsg(ids, body).Times(1)

fcmClient := NewFCMProvider(s.fcmClientMock)
fcmClient.SetMessage(ids, body)
```